libs/pilight/core/common.c

```diff
@@ -234,8 +234,8 @@ int isrunning(const char *program) {
 		return -1;
 	}
 	int i = 0;
-	char name[255], *p = name;
-	memset(&name, '\0', 255);
+	char name[1023], *p = name;
+	memset(&name, '\0', 1023);
 
 	for(i=0;i<psutil_max_pid();i++) {
 		if(psutil_proc_name(i, &p) == 0) {
```

Comment:
I want to suggest to make the buffer length a parameter of the `psutil_proc_name` function as well, because inside that function `read` is called with a buffer of 1024 as well. So, better make the whole stack dynamic:
https://github.com/pilino1234/pilight/blob/330addd253fcb4018f91f1e8278ff0f9af7b0933/libs/pilight/psutil/linux.c#L56